starter-code/package.json

```diff
@@ -3,14 +3,17 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./bin/www"
+    "start": "nodemon --inspect ./bin/www"
```

Comment:
you should not add `--inspect` to the `start` script

if you want to add a script with debugging capabilities, you can also add one for development only

```
 "start-dev": "nodemon --inspect ./bin/www"
```